plugin/builtin/patchtransformer/PatchTransformer.go

```diff
@@ -57,7 +57,7 @@ func (p *plugin) Config(
 	patchJson, errJson := jsonPatchFromBytes(in)
 	if errSM != nil && errJson != nil {
 		err = fmt.Errorf(
-			"unable to get either a Strategic Merge Patch or JSON patch 6902 from %s", p.Patch)
+			"Strategic Merge Patch strategy is failing with: %v, JSON patch 6902 strategy is failing with: %v", errSM, errJson)
```

Comment:
great.

only nit here is that my IDE (goland) will flag this as a style error since the error message starts with a capital letter (https://github.com/golang/go/wiki/CodeReviewComments#error-strings)

suggest just lowercasing "strategic merge patch"